action.yml

```
@@ -81,6 +81,10 @@ inputs:
     default: "false"
     description: |-
       Whether to only run the compilation steps. Default is `false`.
+  comment-fail-only:
+    default: "false"
```

I think we can use "true" as the default value.

```suggestion
    default: "true"
```

---

Changed the default and added a description to the readme file.